### PR TITLE
Add external boolean to Partners table

### PR DESCRIPTION
**Pull request by garyhtou**
closes #1676

run after merging
```ruby
p = Partner.where(slug: 'bank').first
p.external = false
p.save!
```

### CODE REVIEW

File: db/migrate/20210607030039_add_external_to_partners.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddExternalToPartners < ActiveRecord::Migration[6.0]
+  def change
+    add_column :partners, :external, :boolean, null: false, default: true
```
Comment (motdotla) [inline db/migrate/20210607030039_add_external_to_partners.rb:3]: well done here with the default.

can you add a spec to the partner_spec.rb that shows the external defaults to true?

**Review by motdotla — CHANGES_REQUESTED** [review]
one small addition. add a spec to partner_spec.rb showing that external defaults to true

File: db/migrate/20210607030039_add_external_to_partners.rb (hunk 1 above)
Comment (garyhtou) [reply db/migrate/20210607030039_add_external_to_partners.rb:3]: done!

image

**Comment by motdotla** [comment]
merging. i will update Bank partner to false once deployed.

**Pull request merged**